apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -68,38 +68,38 @@ export default class StartNewProject extends React.Component {
         <div style={{clear: 'both'}}/>
 
         {showFullList &&
-        <div>
-          <NewProjectButtons
-            description={i18n.projectGroupPlaylab()}
-            projectTypes={['playlab', 'infinity', 'gumball', 'iceage']}
-          />
-          <NewProjectButtons
-            description={i18n.projectGroupEvents()}
-            projectTypes={['flappy', 'starwarsblocks', 'starwars', 'bounce', 'sports', 'basketball']}
-          />
-          <NewProjectButtons
-            description={i18n.projectGroupArtist()}
-            projectTypes={['artist', 'frozen']}
-          />
-          <NewProjectButtons
-            description={i18n.projectGroupMinecraft()}
-            projectTypes={['minecraft_hero', 'minecraft_designer', 'minecraft_adventurer']}
-          />
-          {canViewAdvancedTools &&
+          <div>
             <NewProjectButtons
-              description={i18n.projectGroupAdvancedTools()}
-              projectTypes={['applab', 'gamelab', 'weblab']}
+              description={i18n.projectGroupPlaylab()}
+              projectTypes={['playlab', 'infinity', 'gumball', 'iceage']}
             />
-          }
-          <NewProjectButtons
-            description={i18n.projectGroupPreReader()}
-            projectTypes={['playlab_k1', 'artist_k1']}
-          />
-          <NewProjectButtons
-            description={i18n.projectGroupMath()}
-            projectTypes={['calc', 'eval']}
-          />
-        </div>
+            <NewProjectButtons
+              description={i18n.projectGroupEvents()}
+              projectTypes={['flappy', 'starwarsblocks', 'starwars', 'bounce', 'sports', 'basketball']}
+            />
+            <NewProjectButtons
+              description={i18n.projectGroupArtist()}
+              projectTypes={['artist', 'frozen']}
+            />
+            <NewProjectButtons
+              description={i18n.projectGroupMinecraft()}
+              projectTypes={['minecraft_hero', 'minecraft_designer', 'minecraft_adventurer']}
+            />
+            {canViewAdvancedTools &&
+              <NewProjectButtons
+                description={i18n.projectGroupAdvancedTools()}
+                projectTypes={['applab', 'gamelab', 'weblab']}
+              />
+            }
+            <NewProjectButtons
+              description={i18n.projectGroupPreReader()}
+              projectTypes={['playlab_k1', 'artist_k1']}
+            />
+            <NewProjectButtons
+              description={i18n.projectGroupMath()}
+              projectTypes={['calc', 'eval']}
+            />
+          </div>
```

Comment:
Github makes it look like I changed a bunch of stuff here, but really I just indented this whole div to fix alignment.